scripts/userstyles.yml

```diff
@@ -447,6 +460,8 @@ userstyles:
   mdn:
     name: MDN
     categories: [development, wiki]
+    icon: mdnwebdocs
+    color: text
```

Comment:
Maybe blue? https://en.wikipedia.org/wiki/MDN_Web_Docs#/media/File:MDN_Web_Docs_logo.svg
Or are we just going off of the Simple Icons colors?

Reply:
See below comments